src/wp-admin/about.php

```diff
@@ -53,172 +53,115 @@
+				<h3><?php _e( 'Color palettes & font sets' ); ?></h3>
+				<p><strong><?php _e( 'Add more design options to any block theme.' ); ?></strong> <?php _e( 'Block theme authors can create unlimited individual color or font sets to offer more specific design options within the same theme. These sets provide more contained design possibilities, allowing for customization without changing the site&#8217;s broader styling, beyond color or typography settings.' ); ?></p>
 			</div>
 			<div class="column is-vertically-aligned-center">
 				<div class="about__image">
-					<img src="https://s.w.org/images/core/6.5/3-style-revisions.webp" alt="" height="436" width="436" />
+					<img src="data:image/svg+xml,%3Csvg width='436' height='436' viewbox='0 0 100 100' xmlns='http://www.w3.org/2000/svg'%3E%3Crect width='100%25' height='100%25' fill='%23ededed' /%3E%3C/svg%3E" alt="" height="436" width="436" />
 				</div>
 			</div>
 		</div>
 
-		<div class="about__section has-3-columns">
-			<div class="column">
-				<div class="about__image">
-					<img src="https://s.w.org/images/core/6.5/4-background-images.webp" alt="" height="270" width="270" />
-				</div>
-				<h3 class="is-smaller-heading" style="margin-bottom:calc(var(--gap) / 4);"><?php _e( 'Do more with background images in Group blocks' ); ?></h3>
-				<p><?php _e( 'Control size, repeat, and focal point options so you can play around with subtle or splashy ways to add visual interest to layouts.' ); ?></p>
-			</div>
-			<div class="column">
+		<div class="about__section has-2-columns">
+			<div class="column is-vertically-aligned-center">
 				<div class="about__image">
-					<img src="https://s.w.org/images/core/6.5/5-cover-aspect-ratio.webp" alt="" height="270" width="270" />
+					<img src="data:image/svg+xml,%3Csvg width='436' height='436' viewbox='0 0 100 100' xmlns='http://www.w3.org/2000/svg'%3E%3Crect width='100%25' height='100%25' fill='%23ededed' /%3E%3C/svg%3E" alt="" height="436" width="436" />
 				</div>
-				<h3 class="is-smaller-heading" style="margin-bottom:calc(var(--gap) / 4);"><?php _e( 'Get more control over images in Cover blocks' ); ?></h3>
-				<p><?php _e( 'Set aspect ratios for Cover block images and easily add color overlays that automatically source color from your chosen image.' ); ?></p>
 			</div>
-			<div class="column">
-				<div class="about__image">
-					<img src="https://s.w.org/images/core/6.5/6-box-shadow.webp" alt="" height="270" width="270" />
-				</div>
-				<h3 class="is-smaller-heading" style="margin-bottom:calc(var(--gap) / 4);"><?php _e( 'Add box shadow supports to even more blocks' ); ?></h3>
-				<p><?php _e( 'With shadow supports enabled, you can create layouts with visual depth or add a little personality to your design.' ); ?></p>
+			<div class="column is-vertically-aligned-center">
+				<h3><?php _e( 'Get more details from your style revisions' ); ?></h3>
```

Comment:
Just a drive-by comment, so apologies if this feedback isn't right, but this appears to be the old heading from WP 6.5? The body of this feature refers to the new layout for navigating and editing pages in the site editor, but the heading here refers to style revisions.

Reply:
From a look at the doc linked in https://github.com/WordPress/gutenberg/issues/62631#issuecomment-2187313214, it looks like the heading is meant to be:

```suggestion
				<h3><?php _e( 'Quick previews for pages' ); ?></h3>
```

Reply:
Good catch, Andrew is right!

Reply:
Aha, yes! this is why I was asking for a second look 🙂